src/integrationTest/resources/docker-compose-solr.yml

```diff
@@ -5,11 +5,18 @@ services:
    image: ${DOCKER_IMAGE}
    restart: unless-stopped
    ports:
-   - 8443
+   - "8443"
    #   - 8000:8000
    #   - 5000:5000
    environment:
-   #   - JMX_ENABLED=true
-   #   - DEBUG=true
-   - GLOBAL_WORKSPACE_solr.suggester.minSecsBetweenBuilds=3
-   # for restoring from backup, check non-ssl variant
+#    - JMX_ENABLED=true
+#    - DEBUG=true
+    - GLOBAL_WORKSPACE_solr.suggester.minSecsBetweenBuilds=3
```

Comment:
do we want to add an integration test for the backup as well maybe? what do you think Hechmi